### DISCUSSION TITLE
Host using SSL?

### DISCUSSION THREAD

**Discussion by chalter-Innovativesol** [Selbi182/SpotifyBigPicture#23] [Q&A]
I host this as a publicly accessible container in AWS. It works perfectly, except it appears to only run using HTTP. Is there a good way to run this using SSL so I can give it my domain's wildcard cert? My main motivation for this is to get Chrome to stop complaining that it's insecure when I load the page.

**Comment by Selbi182** [comment]
Weird, I run it on my private server using HTTPS no problem. What error do you get?
